tests/ops/op_math/test_controlled_op.py

```diff
@@ -659,10 +663,66 @@ def test_sparse_matrix_format(self):
         assert isinstance(lil_mat, sparse.lil_matrix)
 
 
+class TestHelperMethod:
+    """Unittests for the _decompose_no_control_values helper function."""
+
+    def test_crx(self):
+        """Test case with single control wire and defined _controlled"""
+        base = qml.RX(1.0, wires=0)
+        op = Controlled(base, 1)
+        decomp = _decompose_no_control_values(op)
+        assert len(decomp) == 1
+        assert qml.equal(decomp[0], qml.CRX(1.0, wires=(1, 0)))
+
+    def test_inverts_decomp_if_target_inverted(self):
+        """Tests that the method preserves the inverse parameter."""
+        base = qml.RX(1.0, wires=0).inv()
+        op = Controlled(base, 1)
+        decomp = _decompose_no_control_values(op)
+        assert len(decomp) == 1
+        assert decomp[0].inverse
+
+    def test_toffoli(self):
+        """Test case when PauliX with two controls."""
+        op = Controlled(qml.PauliX(2), (0, 1))
```

Comment:
Do we maybe want a test case with non-trivial wire ordering? :thinking:

Reply:
Good idea. I'll add that later.

Reply:
Is this still open to do?